### ISSUE TITLE
Update to Ember Data 3.2, ember-power-select-with-create stopped working as intended

### ISSUE THREAD

**Issue by corrspt** [emberjs/data#5553]
Hi,

I'm sorry if the title is not great, I'm not sure what the problem is. I was using Ember Data 2.18 in my Application (together with Ember 2.18 and ember-power-select-with-create). After updating everything to 3.2 (Ember+CLI+Data) my components using `ember-power-select-with-create` stopped working as expected.

I first thought the problem was with ember and ember-power-select, but after creating a reproduction repo/issue/twiddle for power-select it became apparent the issue is with ember data v3.2 (don't know if above is correct or not, because in Twiddle, I can't seem to use a higher version).

I'm going to paste the content of the issue I posted in ember-power-select:

I'm not sure what's happening but on my application I've upgraded Ember to 3.2 and `power-select-with-create` started behaving weirdly... when I select a value, it will not show in the trigger as the selected value.

I've made a reproduction repo, based on the current master repo of `power-select-with-create`. It's available here [https://github.com/corrspt/ember-power-select-with-create/tree/test-issues](https://github.com/corrspt/ember-power-select-with-create/tree/test-issues). Just clone the repo, run yarn and go to `localhost:4200` (See below for a twidle)

Here's a gif of what happens:

![teste](https://user-images.githubusercontent.com/789263/43474490-f67bd832-94ea-11e8-8774-6a669595bba6.gif)

I've created an Ember-Twiddle for this, The Twiddle is here [https://ember-twiddle.com/417bca3fb3b46432ba255a608a56555e](https://ember-twiddle.com/417bca3fb3b46432ba255a608a56555e). Just switch Ember-Data to version 3.1 or below and things start working.

One thing to note, was that at a time, I was able to user power-select correctly, but not with power-select-with-create, but I'm not sure why, it stopped working to. (the gif shows that).

I'm happy to provide more details if needed

**Comment by runspired** [comment]
Closing as a duplicate of #5555 which is more specific

**Comment by corrspt** [comment]
Hi @runspired Thanks for taking a look at my issue.

On #5555 you mention that `3.2.1` and `3.3.1` fix that issue, but I did try with `3.2.1` on my application and it didn't work (can't remember about `3.3.1`). Also, If I set Ember Data `3.2.1` on Twiddle it also does not work (although since it's not in the options for twiddle, I'm not sure if it's used or not).

So, I'm not sure it's really the same issue.
